src/clims/models/workunit.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import absolute_import
+
+from django.db import models
+from sentry.db.models import (FlexibleForeignKey, sane_repr)
+from clims.models.substance import Substance
+from clims.models.container import Container
+
+
+class WorkUnit(models.Model):
+    """
+    Represents a single item that needs to be processed manually. A WorkBatch consists of 1..n
+    WorkUnits.
+
+    WorkUnits can either point to an external workflow engine (such as Camunda, where they map
+    to UserTasks) or not. If they point to no workflow engine, resolving the WorkUnit will just
+    close that item without further tracking. This makes it possible to use the same views to
+    work on items that are within a workflow or not.
```

Comment:
This is hard to understand without examples.

Reply:
Yes, it's a catch-22. If I send in all the context it might be too much info at once :). But it'll come in the ~next PR

Reply:
ok, that was what we talked about in todays standup :)